### PR TITLE
Added checks for plugin existence

### PR DESCRIPTION
**Pull request by bwisdom88**
I added a silent! on the colorscheme to run silently and ignore changes. I also wrapped everything in if checks. It checks for the vim files of the plugins before it allows those to be used.

The one thing that I dont quite understand is some of the lsp keybinds error out instead of doing nothing. I also havent figured out the tree-sitter loading everytime thing.

### CODE REVIEW

**Comment by bwisdom88** [comment]
Dont have write access to the repo so I cant merge the PR with the other

**Review by erkrnt — APPROVED** [review]
Oh dang! This is great! Will merge this into my working branch now!

**Pull request merged**
